### PR TITLE
feat: introduce group activity management and grading

### PR DESCRIPTION
**Pull request by sjschlapbach**
Before merging, consider the following tasks:
- [x] Do not squash this development branch on merging, since it contains squashed PRs already!
- [x] Remove the "GroupActivityManagement" branch from the cypress-testing.yml file
- [x] Extend test suite with tests related to the group activity grading process
- [x] Extend documentation for group activity creation, student view and grading

### CODE REVIEW

File: packages/graphql/src/services/groups.ts [hunk 1]
```diff
@@ -662,3 +865,246 @@ export async function submitGroupActivityDecisions(
...
+          }
+        }),
+      },
+    },
+  })
+
+  return updatedInstance
+}
+
+export async function finalizeGroupActivityGrading(
```
Comment (rschlaefli) [inline packages/graphql/src/services/groups.ts:1076]: the only part missing here is the assignment of points to the group and participants for passing the activity

File: apps/frontend-manage/src/pages/courses/grading/groupActivity/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+import { Button, H1, H2, UserNotification } from '@uzh-bf/design-system'
+import dayjs from 'dayjs'
+import { GetStaticPropsContext } from 'next'
+import { useTranslations } from 'next-intl'
+import { useRouter } from 'next/router'
+import { useMemo, useState } from 'react'
+import { twMerge } from 'tailwind-merge'
+
+function GroupActivityGrading() {
+  const MAX_POINTS_PER_QUESTION = 25
```
Comment (rschlaefli) [inline apps/frontend-manage/src/pages/courses/grading/groupActivity/[id].tsx]: move out of the component

File: apps/frontend-manage/src/pages/courses/grading/groupActivity/[id].tsx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    )
+
+  if (!groupActivity) {
+    return (
+      <Layout>{t('manage.groupActivity.activityMissingOrNotCompleted')}</Layout>
+    )
+  }
+
+  // sort invalid submissions last and graded ones to the back
+  const submissions = [...(groupActivity.activityInstances || [])].sort(
```
Comment (rschlaefli) [inline apps/frontend-manage/src/pages/courses/grading/groupActivity/[id].tsx]: sorting in useMemo

File: packages/graphql/src/services/groups.ts (hunk 1 above)
Comment (sjschlapbach) [reply packages/graphql/src/services/groups.ts:1076]: has been completed in the meantime

**Review by rschlaefli — APPROVED** [review]
LGTM

**Pull request merged**